src/tiff.ml

```diff
@@ -960,9 +991,20 @@ module Data = struct
 end
 
 (* have to specify all 4 or else it defaults to whole file*)
-let data (type repr kind) ?window t (f : File.ro)
+let data (type repr kind) ?plane ?window t (f : File.ro)
```

Comment:
```suggestion
let data (type repr kind) ?(plane=0) ?window t (f : File.ro)
```

I think this saves us an allocation and a bit of logic below :)

Reply:
I went the other way on this, and leant into it being an option more - I now require that you specify for planar data - I don't think there should be a default plane, that'll lead to confusion when someone forgets. I added some more tests around this too.

Reply:
That's perfect, makes more sense now thank you